### PR TITLE
Recreate /newsletter

### PR DESCRIPTION
**Pull request by michaltrzesimiech**
Closes https://github.com/rtcharity/eahub.org/issues/520

### CODE REVIEW

File: eahub/config/urls.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     path('groups/', views.groups, name='groups'),
     path('admin/', admin.site.urls, name='admin'),
     path('about/', views.about, name='about'),
+    path('newsletter/', TemplateView.as_view(template_name='eahub/newsletter.html'), name='newsletter'),
```
Comment (taymonbeal) [inline eahub/config/urls.py]: We shouldn't have the same content at two different URLs (and we _can't_ have two URL patterns in Django with the same name; that will break things). One of these should redirect to the other.

Comment (taymonbeal) [reply eahub/config/urls.py]: Did you mean for one of these to be a redirect?

Comment (michaltrzesimiech) [reply eahub/config/urls.py]: Yeah, I made a dumb guess. I've read In here on how to do this properly (I think): https://stackoverflow.com/questions/42614172/how-can-i-redirect-from-view-in-django/42614265, but I guess I'll hold on until I have a local environment running.

Comment (taymonbeal) [reply eahub/config/urls.py]: Use the `LegacyRedirectView`; we've got a bunch of these elsewhere in the file.

Comment (michaltrzesimiech) [reply eahub/config/urls.py]: Lol, you're right. Done that.

**Pull request merged**